doc/tech/approach.md

```diff
@@ -3,97 +3,230 @@
 
-<a name="5.1.1"></a>
-### 5.1.1 Executive Summary
+•	ETSI NFV model does not explicitly call out an HW Abstraction layer on top of the HW resources and by that many SW Virtualisation Layers act upon the physical HW components in a non-portable way that do not allow an HW Infrastructure to compose abstracted HW resources from pools of individual HW components
+
+•	ETSI NFV lack of spelled out HW Abstraction does not enable a clear way to abstract and describe the technical possibilities that HW layer networking separation, encapsulation, acceleration, etc. can be implemented either in a HW Infrastructure Managed SmartNIC on the Server units or in the shared underlay switching fabric i.e. on the physical Switching Units 
+
+•	ETSI NFV model does not include any reference to SDN (Software Define Network) controller(s) and relevant integration points into NFVI and MANO 
+
+•	ETSI NFV model does not have a way to enable programmable forwarding planes in the HW layer controlled from higher layers of virtualization managers, orchestrators or Network Functions
+
+<p align="center"><img src="../tech/figures/ETSI.png"><b> Figure 5-1-1-1: ETSI model and Networking.</b></p>
+
+Note: Other areas of concern may be identified and addressed during the later stages of work on the RM/RA contents for Networking.
+
+<a name="5.1.1.2"></a>
+#### 5.1.1.2 Specific topics to be included in the Networking Scope
+To address the concern areas detailed above the specific topics listed below will be covered under Networking. The list below does not pretend to be complete. It will be a subject to modifications as a result of the community discussion.
+
+•	Network connectivity for L3 tenants, GWs, SDS, etc
+
+•	Spine-Leaf Architecture ( 3 Tier or 5 Tier CLOS architecture )
+
+•	Automation of the programmable newtorking fabric
+
+•	Variety of Underlay protocols that can be used as a transport layer within the physical fabric
+
+•	Variety of Overlay protocols
+
+•	SDN integration with Infrastructure Management
+
+•	SRIOV workload interworking
+
+•	DPDK workload interworking
+
+•	Service Function Chaining (SFC ) and how it can be achieved
+
+•	New networking realisation technologies ( SmartNIC , FPGA ,..)
+
+•	Inter K8s POD connectivity and CNI
+
+•	Networking aspects of migration from IaaS to CaaS (from the VNF paradigm to the CNF paradigm)
+
+•	Coexistence of IaaS and CaaS and protracted parallel operations
+
+•	Version controlled catalog of APIs, and their respective spans of control, capabilities and purpose, to facilitate predictable integration with a wide selection of fabric implementations
 
-> _Placeholder for the Executive Summary (targeting Baldy for first draft). **If you are interested in writing this content, please contact the RM lead.**_
```

Comment:
@TFredberg Are we still going to scope in driving a SmartNIC ABI standard? If so, let's declare it in the scope.

Reply:
@markshostak Agree that this could be included e.g. in section "5.1.1.2 Specific topics to be included in the Networking Scope".
Could possibly extend the sub bullet "New networking realisation technologies ( SmartNIC , FPGA ,..)" with the text "and drive relevant industry standard interfaces to the Cloud Infrastructure components and VNFs/CNFs refence points.

Reply:
@TFredberg Great! Then let's make sure it gets added to the PR. -M

Reply:
We decided on the NW Focus Group that Walter will inlcude it in the text as one of the targets, but likely in a bit more broader statement.